aten/src/ATen/native/Resize.h

```diff
@@ -94,6 +94,11 @@ inline void setStrided(
 
   /* size and stride */
   AT_ASSERT(size.size() == stride.size());
+
+  for(auto val : stride){
+    TORCH_CHECK(val >= 0, "Negative strides are not supported at the moment");
```

Comment:
You should add a test to  test_torch.py that checks the error message is raised. Here is an example: https://github.com/pytorch/pytorch/blob/4ef854b4b4108fa03d9d5081d4bf25bdf0717252/test/test_torch.py#L374